src/components/controls/dates/Calendar/Calendar.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { expect } from "@storybook/jest"
+import { ComponentMeta, ComponentStory } from "@storybook/react"
+import { userEvent, within, waitFor } from "@storybook/testing-library"
+import { useState } from "react"
+import { Calendar } from "./Calendar"
+
+export default {
+  title: "Controls/Dates/Calendar",
+  component: Calendar,
+  argTypes: {
+    disabled: {
+      control: "boolean",
+    },
+  },
+} as ComponentMeta<typeof Calendar>
+
+const Template: ComponentStory<typeof Calendar> = (args) => {
+  const { value: defaultValue } = args
+  const [value, setValue] = useState<Date | null>(defaultValue || null)
+  return <Calendar {...args} value={value} onChange={setValue} />
+}
+
+export const Default = Template.bind({})
+Default.args = {}
+
+Default.play = async ({ args }) => {
+  expect(args.value).toBeFalsy()
+}
+
+export const DefaultValue = Template.bind({})
+DefaultValue.args = {
+  value: new Date(new Date().setDate(10)),
+}
+
+DefaultValue.play = async ({ canvasElement, args }) => {
+  const canvas = within(canvasElement)
+  expect(args.value).toBeTruthy()
+
+  const currentDayButton = canvas.getByRole("button", { name: "10" })
+  expect(currentDayButton.getAttribute("data-selected")).toBe("true")
+}
+
+export const Mouse = Template.bind({})
+Mouse.args = {
+  value: new Date(new Date().setDate(10)),
+}
+
+Mouse.play = async ({ canvasElement, args }) => {
```

Comment:
I have sticked to grouping the template, args and interaction tests without empty lines between to make it easy to see the full "story". What do you think about doing that here as well?

```suggestion
export const Mouse = Template.bind({})
Mouse.args = {
  value: new Date(new Date().setDate(10)),
}
Mouse.play = async ({ canvasElement, args }) => {
```